spec/controllers/concerns/billable_event_trackable_spec.rb

```diff
@@ -21,6 +21,14 @@
   let(:ial_context) { IalContext.new(ial: 1, service_provider: current_sp) }
   let(:request_id) { SecureRandom.hex }
   let(:session_started_at) { 5.minutes.ago }
+  let(:active_profile) do
+    create(
+      :profile, :active, :verified, user: current_user,
+                                    initiating_service_provider: current_sp
+    )
```

Comment:
personal style, I would line up all the args
```suggestion
    create(
      :profile,
      :active,
      :verified,
      user: current_user,
      initiating_service_provider: current_sp,
    )
```